Errors building extension against Theia 0.3.16

Trying to build an extension against 0.3.16 for the first time (currently builds against 0.3.14), I'm getting the following errors:
```
$ yarn run clean && yarn run build
yarn run v1.12.3
$ rimraf lib
Done in 0.49s.
yarn run v1.12.3
$ tsc
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(135,28): error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(135,32): error TS2304: Cannot find name 'Ajv'.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(153,58): error TS1183: An implementation cannot be declared in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(156,74): error TS1183: An implementation cannot be declared in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(161,103): error TS1183: An implementation cannot be declared in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(162,36): error TS1039: Initializers are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(163,34): error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(165,31): error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(166,25): error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(167,13): error TS1036: Statements are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(168,17): error TS1036: Statements are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(171,17): error TS1036: Statements are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(176,103): error TS1183: An implementation cannot be declared in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(178,37): error TS1039: Initializers are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(179,13): error TS1036: Statements are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(180,30): error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(181,17): error TS1036: Statements are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(182,21): error TS1036: Statements are not allowed in ambient contexts.
node_modules/@theia/workspace/lib/browser/workspace-service.d.ts(185,21): error TS1036: Statements are not allowed in ambient contexts.
error Command failed with exit code 2.
```

--- a/packages/workspace/src/browser/workspace-service.ts
+++ b/packages/workspace/src/browser/workspace-service.ts
@@ -498,12 +498,12 @@ export interface WorkspaceInput {
 
 }
 
-interface WorkspaceData {
+export interface WorkspaceData {
     folders: Array<{ path: string }>;
     // TODO add workspace settings settings?: { [id: string]: any };
 }
 
-namespace WorkspaceData {
+export namespace WorkspaceData {
     const validateSchema = new Ajv().compile({
         type: 'object',
         properties: {

